tests/rptest/services/admin.py

```diff
@@ -388,6 +388,29 @@ def get_node_config(self):
     def get_features(self, node=None):
         return self._request("GET", "features", node=node).json()
 
+    def supports_feature(self, feature_name: str, nodes=None):
+        """
+        Returns true whether all nodes in 'nodes' support the given feature. If
+        no nodes are supplied, uses all nodes in the cluster.
+        """
+        if not nodes:
+            nodes = self.redpanda.nodes
+
+        def node_supports_feature(node):
+            features_resp = None
+            try:
+                features_resp = self.get_features(node=node)
+            except RequestException:
+                return False
+            features_dict = dict(
+                (f["name"], f) for f in features_resp["features"])
+            return features_dict[feature_name]["state"] == "active"
+
+        for node in nodes:
```

Comment:
nit: `return not any([not_supports_feature(node) for node in nodes])`

Reply:
I actually prefer the current syntax -- it seems easier to grok (likely people are more familiar with procedural code style vs functional code style)